pkgs/development/python-modules/ansible-lint/default.nix

```diff
@@ -42,7 +44,9 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     ansible-compat
     ansible-core
+    black
```

Comment:
Is not black a runtime dependency, too?!

Reply:
Yes, it is. That's why its propagated.